ecaudit/src/test/java/com/ericsson/bss/cassandra/ecaudit/facade/TestDefaultAuditor.java

```diff
@@ -114,14 +114,16 @@ public void testAuditFilteredThrowsException()
     {
         AuditEntry logEntry = AuditEntry.newBuilder().build();
         when(mockFilter.isWhitelisted(logEntry)).thenThrow(new ReadTimeoutException(ConsistencyLevel.QUORUM, 1, 1, false));
+        when(mockObfuscator.obfuscate(logEntry)).thenReturn(logEntry);
```

Comment:
The name of this test case should be updated - testAuditFilteredThrowsException()
It now handles the exception... So maybe call the method "testAuditFilteredHandlesException" or something similar...

Reply:
Updated method name